### DISCUSSION TITLE
歌手列表右边的🔧图标是什么意思

### DISCUSSION THREAD

**Discussion by mysteryzzp** [gitbobobo/StreamMusic#528] [Q&A] [answered]
![Screenshot_20240829_194611_cn_aqzscn_stream_music_MainActivity.jpg](https://github.com/user-attachments/assets/3f62ea0f-e236-4b78-b6d7-18ef0866c3fc)

**Comment by gitbobobo** [comment, marked as answer]
可能存在重复歌曲
